File: secondLargest.java
import java.util.*;
class secondLargest
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        int a,b,c,s,l,sl;
        System.out.println("Enter three number: ");
        a=sc.nextInt();
        b=sc.nextInt();
        c=sc.nextInt();
        l=Math.max(a,Math.max(b,c));
        s=Math.min(a,Math.min(b,c));
        sl=(a+b+c)-(l+s);
        System.out.println("Second Largest number is: "+sl);
    }
}

File: second largest.c
int main()
{
    int a,b,c;
    printf("Enter the three numbers: \n");
    scanf("%d%d%d",&a,&b,&c);
    if(a>b&&a>c)
        if(b>c)
        printf("b is Second largest no.: %d",b);
    else
        printf("c is Second largest no.: %d",c);
    if(b>a&&b>c)
        if(a>c)
        printf("a is Second largest no.: %d",a);
    else
        printf("c is Second largest no.: %d",c);
    if(c>a&&c>b)
        if(b>a)
        printf("a is Second largest no.: %d",a);
    else
        printf("b is Second largest no.: %d",b);
    return 0;
}
